test/rest.test.js

```diff
@@ -1571,6 +1571,41 @@ describe('strong-remoting-rest', function() {
           .end(expectErrorResponseContaining({ message: 'an error' }, done));
       });
 
+      it('should return 500 if an array of errors is thrown', function(done) {
+        var testError = new Error('expected test error');
+        var errArray = [testError, testError];
+
+        var expectedMessage =
+          'Failed with multiple errors, see `details` for more information.';
+
+        function method(error) {
+          return givenSharedStaticMethod(function(cb) {
+            cb(error);
+          });
+        }
+
+        request(app).get(method(testError).url)
+          .set('Accept', 'application/json')
+          .expect(500)
+          .end(function(err, res) {
+            if (err) return done(err);
+            expectedDetail = res.body.error;
+            delete expectedDetail.statusCode;
+
+            request(app).get(method(errArray).url)
+            .set('Accept', 'application/json')
```

Comment:
I think this line (and the following two lines should be indented by one level).

```js
request(app).get(...)
  .set(...)
  .expect(500)
  .end(...);
```